service/src/main/kotlin/no/nav/su/se/bakover/service/personhendelser/OppgavebeskrivelseMapper.kt

```diff
@@ -0,0 +1,22 @@
+package no.nav.su.se.bakover.service.personhendelser
```

Comment:
Jeg synes denne ikke hører hjemme i domene/service-laget. Dette er vel en OppgaveClient only ting?

Reply:
Kan flytte 👍

Reply:
Eller hmm, den brukes for å lage `OppgaveConfig` som er ett domæneobjekt, er det ikke greit att den er här da?

Reply:
Jeg tror ikke OppgaveConfig bør være i domenet da! Har stusset over det før.

Reply:
Pratet med Aryan om denne i går. Tror vi ble enige om å ikke ha beskrivelsesstrengen i OppgaveConfig, men heller ha Hendelsestypen. Da kan vi flytte mappingen som foreslått.

Reply:
done!